packages/client/src/forms/utils.ts

```diff
@@ -652,7 +652,7 @@ export const convertToMSISDN = (phone: string) => {
    */
   const countryCode =
     window.config.COUNTRY.toUpperCase() === 'FAR'
-      ? 'ZMB'
+      ? 'BGD'
```

Comment:
@euanmillar Couldn't we have changed the value of country in here: https://github.com/opencrvs/opencrvs-farajaland/blob/a962aeb1f32a9a470d7a183831efab7fa784c1b5/src/client-configs/client-config.prod.js#L19. That way we won't need this custom core image?

Reply:
I pushed a commit to the Farajaland branch with updated COUNTRY value and it seems to be working without this custom core image

Reply:
@Zangetsu101 we need this because, if Syeda wants to pretend to be an informant, she will be putting her Bangladesh phone number into the declaration form.  When the form is submitted, it will change the country code to a Zambian code and she wont get the notification SMS messages she is trying to test to see if they can be configured